docs/application/web/api/4.0/device_api/tv/index.html

```diff
@@ -176,19 +176,19 @@ <h4 id="TV Controls">TV Controls</h4>
 <td>No</td>
 </tr>
 <tr><td><a href="tizen/tvinfo.html">TV Information</a></td>
-<td>This API allows receiving key events generated when the user presses a key of an Input Device (for example a TV remote control).</td>
+<td>This API provides interfaces and methods for getting information about TV settings.</td>
 <td>2.4</td>
 <td>Optional</td>
 <td>No</td>
 </tr>
 <tr><td><a href="tizen/tvinputdevice.html">TV Input Device</a></td>
-<td>This API provides interfaces and methods for control of TV Window (e.g. main window, PIP window).</td>
+<td>This API allows receiving key events generated when the user presses a key of an Input Device (for example a TV remote control).</td>
 <td>2.3</td>
 <td>Optional</td>
 <td>Yes</td>
 </tr>
 <tr><td><a href="tizen/tvwindow.html">TV Window</a></td>
-<td>This API provides interfaces and methods for getting information about TV setting.</td>
+<td>This API provides interfaces and methods for control of TV Window (e.g. main window, PIP window).</td>
```

Comment:
This API provides interfaces and methods to control the TV Window. For example, the main window and PIP window.

Reply:
@pkosko, gentle reminder to fix the comments.